src/main/java/assets/css/custom.css

```diff
@@ -176,6 +176,20 @@ text-align:center;
     border: 3px dashed rgba(178, 103, 253, 1);
 }
 
+#opponent.laser td:hover {
+    background-color: rgba(73, 250, 15, 0.4);
```

Comment:
clear css, we may want to clear some of the styles up since they are a little convoluted